language/src/modules/Language.rb

```diff
@@ -1402,7 +1402,8 @@ def fbiterm?
     #
     # @return [Boolean] true if it's read-only; false otherwise.
     def readonly
-      ProductFeatures.GetBooleanFeature("globals", "readonly_language")
+      @readonly unless @readonly.nil?
+      @readonly = ProductFeatures.GetBooleanFeature("globals", "readonly_language")
```

Comment:
is it needed? How big impact does it have? BTW this is not reset in main, so exactly problem with dirty value I point in previous commit.

Reply:
> is it needed? How big impact does it have?

Only for consistency with the same changes in the Timezone module

> BTW this is not reset in main

Arrrg! I forgot to add the `@readonly = nil` statement in #main.